docs/sphinx/_ext/dagster-sphinx/dagster_sphinx/__init__.py

```diff
@@ -83,6 +83,18 @@ def check_public_method_has_docstring(env: BuildEnvironment, name: str, obj: obj
         record_error(message)
 
 
+# Note that in our codebase docstrings with attributes will usually be written as:
+#
+#     Attributes:
```

Comment:
Is this backwards? Should we say we use `Args` in our codebase?

Reply:
Nah this is the code that checks for attributes and throws an error if it gets detected

Reply:
Ah okay, cool. 👍